docs/.vuepress/components/AbsoluteLayoutDoc.vue

```diff
@@ -0,0 +1,194 @@
+<template>
```

Comment:
It seems like linter did not process this file. 
Could you please run lint for this file to fix indentation and other styling issues.

Reply:
linter did not return any errors, nor made any changes